WooCommerce/Classes/Authentication/Navigation Exceptions/NotWPErrorViewModel.swift

```diff
@@ -0,0 +1,65 @@
+import UIKit
+import SafariServices
+import WordPressAuthenticator
+import WordPressUI
+
+
+/// Configuration and actions for an ULErrorViewController, modelling
```

Comment:
typo: `modelling` -> `modeling`
```suggestion
/// Configuration and actions for an ULErrorViewController, modeling
```